installers/ansible/roles/pgo-operator/tasks/namespace.yml

```diff
@@ -14,13 +14,12 @@
     target_ns_script: "{{ lookup('template', 'add-targeted-namespace.sh.j2') }}"
   with_items: "{{ nslist.split(',') | map('trim') | list }}"
   when:
-    - namespace_mode != 'dynamic'
-    - dynamic_rbac == 'false'
+    - reconcile_rbac == 'false'
```

Comment:
The idea here is that anytime RBAC reconciliation is disabled (i.e. `reconcile_rbac == 'false'`) regardless of namespace mode, then all RBAC resources required by the Operator will need to be manually created in each target namespace, so we go ahead and run `add-targeted-namespace.sh` in each space when installing.

And if `reconcile_rbac` is set to `true` and reconciliation is enabled, then we do not create all of the required RBAC using `add-targeted-namespace.sh`, but rather execute the tasks below to assign the Operator the RBAC it needs into those target namespaces to reconcile its required RBAC (and more specifically, these tasks are executed when namespace mode is `readonly` or `disabled` - otherwise a ClusterRole is created when the namespace mode is `dynamic`).